openwisp_radius/api/views.py

```diff
@@ -335,6 +336,8 @@ class RegisterView(RadiusTokenMixin, DispatchOrgMixin, BaseRegisterView):
 
     def get_response_data(self, user):
         data = super().get_response_data(user)
+        # create a RegisteredUser object for every user that registers through API
+        RegisteredUser.objects.create(user=user)
```

Comment:
Not convinced of this.
How do we know which identity verification method the user is registering with?
Maybe it makes sense to add this to the registration serializer, so the user will select the method from the captive page and we'll get it here without the need of adding complicated code, what do you think?

Reply:
Done (https://github.com/openwisp/openwisp-radius/pull/237/commits/f08a1ad5ef672c1d1120682b6f585c551ee2ee42)
Now I need to write some unit tests... working on it.